src/librustc/ty/context.rs

```diff
@@ -1204,16 +1206,18 @@ direct_interners!('tcx,
 
 macro_rules! slice_interners {
     ($($field:ident: $method:ident($ty:ident)),+) => (
-        $(intern_method!('tcx, $field: $method(&[$ty<'tcx>], alloc_slice, Deref::deref,
-                                               |xs: &[$ty]| -> &Slice<$ty> {
-            unsafe { mem::transmute(xs) }
-        }, |xs: &[$ty]| xs.iter().any(keep_local)) -> Slice<$ty<'tcx>>);)+
+        $(
+            intern_method!('tcx, $field: $method(&[$ty<'tcx>], alloc_slice, Deref::deref,
+                                                   |xs: &[$ty]| -> &Slice<$ty> {
+                unsafe { mem::transmute(xs) }
+            }, |xs: &[$ty]| xs.iter().any(keep_local)) -> Slice<$ty<'tcx>>);
+        )+
```

Comment:
Does anything about this need to change? I think it's just leftovers from going back & forth.